server/store.go

```diff
@@ -66,35 +66,52 @@ func (s *storeInfo) downTime() time.Duration {
 	return time.Since(s.stats.LastHeartbeatTS)
 }
 
+func (s *storeInfo) leaderCount() uint64 {
+	return uint64(s.stats.LeaderCount)
+}
+
 func (s *storeInfo) leaderRatio() float64 {
-	if s.stats.TotalRegionCount == 0 {
+	return float64(s.stats.LeaderCount)
```

Comment:
We don't have any other factors to calculate for now, so I just return the leader count.
How about rename to `leaderScore`?